src/review/ios/index.js

```diff
@@ -1,44 +1,51 @@
 const { CronJob } = require('cron')
 
 const client = require('../../common/glip')
-const engine = require('../common/nunjucks')
+const { engine, PRESET_APPS } = require('../common')
 const { loadDb, saveDb } = require('./db')
-const { getReviews } = require('./spider')
+const { getReviews, getFlattenReviews } = require('./spider')
 const { compareReviews, mergeReviews } = require('./util')
 
-const RINGCENTRAL_APPS = {
-  glip: 715886894,
-  ringcentral: 293305984,
-  meetings: 688920955
-}
-
 const db = loadDb()
+const IOS_PRESET_APPS = PRESET_APPS.ios
 
 const monitors = {}
 const notifyReview = (group, app, number, isNew) => {
   const entry = db[group][app].reviews[number - 1]
   const review = {
     title: entry.title.label.trim(),
     stars: parseInt(entry['im:rating'].label.trim()),
-    content: entry.content.label.split('\n').map((line) => '> ' + line).join('\n'),
+    content: entry.content.label
+      .split('\n')
+      .map(line => '> ' + line)
+      .join('\n'),
     author: entry.author.name.label
   }
-  const message = engine.render('ios/review.njk', { number, review, name: db[group][app].name, new: isNew })
+  const message = engine.render('ios/review.njk', {
+    number,
+    review,
+    name: db[group][app].name,
+    new: isNew
+  })
   client.post(parseInt(group), message)
 }
 const cronJob = async (group, app) => {
-  let reviews = await getReviews(app)
-  reviews = reviews.slice(1)
-  const delta = compareReviews(db[group][app].reviews, reviews)
-  db[group][app].reviews = mergeReviews(db[group][app].reviews, reviews)
-  delta.forEach((number) => {
-    notifyReview(group, app, number, true)
-  })
-  saveDb(db)
+  let reviewCollections = await getReviews(app)
+  reviewCollections.forEach(i => i && i.shift())
+  var oldReviews = db[group][app].reviews
+  for (let index in reviewCollections) {
+    let reviews = reviewCollections[index]
+    const delta = compareReviews(oldReviews, reviews)
```

Comment:
`oldReviews` are multiple collections merged together.
`reviews` is data from only one collection. 
Can you compare them?

Reply:
OK, I read the code of `compareReviews` again and I think you can compare them.